### ISSUE TITLE
The Navbar is not responsive when resizing the browser window.

### ISSUE THREAD

**Issue by mayura-andrew** [sef-global/scholarx-frontend#73]
**Describe the bug**
 The Navbar is not responsive when resizing the browser window. The Navbar does not adapt to the changing window size.

**To Reproduce Steps to reproduce the behavior:**

1. Open the website in a web browser.
2. Resize the browser window.
3. Observe the Navbar.

**Expected behavior**
 The Navbar should be responsive and adapt to the size of the browser window. As the window size decreases, the Navbar should collapse into a hamburger menu or similar.

Screenshots If applicable, add screenshots to help explain your problem.
[Screencast_20240114_130504-1.webm](https://github.com/sef-global/scholarx-frontend/assets/48531182/cb458023-6186-4212-98cb-062be960994e)

Desktop (please complete the following information):

OS: Fedora Linux 39 (KDE Plasma) x86_64 
Browser:  Mozilla Firefox 
Version: 121.0

Additional context The issue occurs when resizing the browser window on a desktop.

**Comment by mayura-andrew** [comment]
[Screencast_20240114_130504-1.webm](https://github.com/sef-global/scholarx-frontend/assets/48531182/a845519d-0874-4246-8ef5-b11c7a668e84)

**Comment by mayura-andrew** [comment]
[Screencast_20240114_130504-1.webm](https://github.com/sef-global/scholarx-frontend/assets/48531182/c8550632-f593-4f32-a4f7-05b163a2bc41)

here the issue

**Comment by vishwaikon** [comment]
Hi, Can I please work on this issue ?

**Comment by dilankavishka** [comment]
Hi @anjula-sack @mayura-andrew Can i work on this issue please?

**Comment by mayura-andrew** [comment]
Hi @vishwaikon go head 
hey @dilankavishka can you work on another issue. Thank you:)

**Comment by kavinda-100** [comment]
https://github.com/sef-global/scholarx-frontend/assets/152590601/77970498-c0b5-4942-aaff-7fdf4cfc5346

refactor existing css with tailwindcss utility class.

**Comment by mayura-andrew** [comment]
@vishwaikon hey any updates ?

**Comment by mayura-andrew** [comment]
> 2024-01-22-13-25-07.mp4 
> refactor existing css with tailwindcss utility class.

nice. can you keep this changes in separate branch ?

**Comment by vishwaikon** [comment]
@mayura-andrew  Hi, I figured out an issue the nav icon has with mobile devices where the icon gets positioned in the centre and fixed it. But yet couldn't fix the issue with the navbar on large screens. I'll try to learn and figure it out and will update the progress. Good job @kavinda-100  !.  Regards!

**Comment by anjula-sack** [comment]
Hi @vishwaikon, Shall we merge the @kavinda-100's update since it's already done? You can work on another issue. @mayura-andrew Will you be able to find another issue for @vishwaikon?

**Comment by kavinda-100** [comment]
> > 2024-01-22-13-25-07.mp4
> > refactor existing css with tailwindcss utility class.
> 
> nice. can you keep this changes in separate branch ?

ok!. I will create new branch for keep these changes

**Comment by mayura-andrew** [comment]
@anjula-sack Yeah, sure. I will find an issue for @vishwaikon and assign it if he's interested!

**Comment by mayura-andrew** [comment]
@vishwaikon hey ! https://github.com/sef-global/scholarx-backend/issues/94 would you able to work on this issue?

**Comment by vishwaikon** [comment]
@mayura-andrew , I'll work on it.

**Comment by mayura-andrew** [comment]
@vishwaikon nice 👍 go head

**Comment by anjula-sack** [comment]
@mayura-andrew Is this issue still valid?

**Comment by mayura-andrew** [comment]
https://github.com/user-attachments/assets/00574234-8adf-4912-a4a0-656117a69437

This needs to be fixed, right? @anjula-sack

**Comment by YashmithaDilshan27** [comment]
@mayura-andrew  Is that issue still open, or has it been resolved?

**Comment by Janani-Balasooriya** [comment]
Hi @mayura-andrew @anjula-sack ! 

Is this issue still available? Can I still work on this?

In the navbar now it has a notification banner and it overlaps with the hero section content.

![Image](https://github.com/user-attachments/assets/4cfa1cb0-eb36-44cb-bfff-33e87213c5f9)

![Image](https://github.com/user-attachments/assets/59861acd-fb26-4212-8f64-1c1c7b70f601)

**Comment by anjula-sack** [comment]
You can fix it, if you like @Janani-Balasooriya

**Comment by Janani-Balasooriya** [comment]
Thank you, @anjula-sack ! Will do asap. 🚀

**Comment by Janani-Balasooriya** [comment]
Hey @anjula-sack @mayura-andrew , whenever you get a chance, could you please take a look at my pull request?

Thanks so much!
